web/packages/extension/tools/sign_xpi.js

```diff
@@ -24,7 +24,8 @@ async function sign(
     }
 
     if (result.downloadedFiles.length === 1) {
-        fs.renameSync(result.downloadedFiles[0], destination);
+        await fs.promises.copyFile(result.downloadedFiles[0], destination);
```

Comment:
Why is that?

Reply:
`renameSync` fails if the destination is on a different drive (hit this on my local PC).

Reply:
Hmm, since this is not trivial, maybe we should add a comment about it?